### ISSUE TITLE
Attribute values cannot be made executable

### ISSUE THREAD

**Issue by ghost** [dxw/Fammel#2]
They can only contain literals. Also:
- Attribute lists cannot span multiple lines
- Have no built-in shortcuts like `:selected => false` instead of `:selected => "selected"`.

**Comment by Kerrick** [comment]
Just curious: What are the odds that this will be fixed soon? I noticed the project hasn't had committed code since 2009.

**Comment by dxw** [comment]
Pretty much zero, I think -- We've more or less abandoned the idea of WordPress/haml. Sorry!

**Comment by Kerrick** [comment]
Shame... Do you know of any other HAML solutions for PHP as good as Fammel, but with executable attribute values?

**Comment by dxw** [comment]
No, I'm afraid not. It's quite difficult to do in PHP. You can cheat by using a load of regexes to implement it, but that's very bug-prone and hard to maintain -- we wanted to do it with a proper grammar, which is why we started working on Fammel. But executable attributes means you suddenly have to include a large amount of the PHP grammar in with the haml grammar, which made the whole thing a lot more complicated.

We also decided that our clients might actually not like us to use haml, as it would make it harder for other devs to take over our work should they decide to move on from us.

So -- we more or less binned it. But I'd be delighted to see someone take it over!
